Gemfile.lock

```diff
@@ -116,7 +116,7 @@ GEM
     faraday_middleware (1.2.0)
       faraday (~> 1.0)
     fastimage (2.2.6)
-    fastlane (2.205.2)
+    fastlane (2.210.1)
```

Comment:
the actual change of note here

Reply:
Shouldn't the version be specified in the actual Gemfile? Right now it's just `gem "fastlane"`

Reply:
It can be, if we want to fix a specific version there. Right now we just get latest when running `bundle update fastlane` and set in the lockfile. Seems worth having explicit in the Gemfile to me though. Will update.